pkg/controller/controller.go

```diff
@@ -34,20 +34,23 @@ var (
 )
 
 func Start() error {
-	if !bpfConfig.AdsEnabled() && !bpfConfig.WdsEnabled() {
-		return fmt.Errorf("controller start failed")
-	}
+	if bpfConfig.EnableByPass {
+		client = NewXdsClient()
```

Comment:
client should be declared outside bypass

Reply:
Done